src/utils/parser.py

```diff
@@ -0,0 +1,60 @@
+import spacy
+import en_core_web_sm
+import re
+nlp = en_core_web_sm.load()
+
+'''
+Parser functions are defined below
+'''
+
+def time_parser(ents):
+    return [ent for ent in ents if ent.label_ == 'TIME']
+
+def date_parser(ents):
+    return [ent for ent in ents if ent.label_ == 'DATE']
+
+def location_parser(message):
+    
+    
+    places_list = ['SJT','TT','SMV'] #List of possible location meetings
+
```

Comment:
Don't hardcode values, we can have meeting anywhere if you are considering VIT. Include NER for location here for detecting location and time from messages.

Reply:
The NER for location performs very poorly, the Time and Date NER works well, thats why I left out the NER for location. Moreover I think having a list of locations might help users to customize this for their specific use

Reply:
Use RASA's context model here for identifying the entities like location date and time. You can also try spacy models for the same. If you go with specific locations , we need to consider other locations which are outside vit.